src/mod.ts

```diff
@@ -36,6 +36,12 @@ const run = async () => {
 
   options.log(options);
 
+  // Clean output directory
+  if (options.outputDirectory) {
+    await Deno.remove(options.outputDirectory, { recursive: true })
+    options.log(`remove directory: ${options.outputDirectory}`);
+  }
+
```

Comment:
We only clean the `outputDirectory` when writing to the filesystem - not when displaying the manifest to STDOUT.

Is there a better way to check when we're writing to the filesystem? I based this approach off [a similar check here](https://github.com/slackapi/deno-slack-builder/blob/main/src/mod.ts#L48-L52).